docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -286,21 +288,30 @@ as long as each module has a unique ESC index. The light IDs on each module are
 Refer to the `uavcan.equipment.indication.LightsCommand section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#lightscommand>`_ in the specification for more details.
 
 Service Requests
-*****************
+========================
 Service requests are messages sent to a specific target node from another node, and to which the sending node expects to receive a response message.
 
 uavcan.protocol.GetNodeInfo (Data Type ID = 1)
-################################################
+---------------------------------------------------------------
 This request has no payload fields. The response message from the receiving node should include the status of the node as defined by the NodeStatus message, the 
 software and hardware version of the node, and the name of the node. For Vertiq modules, the name of each node is currently “iq_motion.esc”.
 
 Refer to the `uavcan.protocol.GetNodeInfo section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getnodeinfo>`_ in the specification 
 for more details
 
+uavcan.protocol.dynamic_node_id.Allocation (Data Type ID = 1)
```

Comment:
I swear it's data type ID 1 also
![image](https://github.com/user-attachments/assets/3b2f7104-dabb-49d8-8577-19c0a9db9a31)